src/utils/breakpoints.js

```diff
@@ -0,0 +1,14 @@
+const minSM = 640
+const minMD = 768
+const minLG = 1024
+const minXL = 1280
```

Comment:
**question**: What are these for? I didn't see them being used.

Reply:
Ah, they're used in a set of scrollytelling components I built for individual articles that I haven't PR-ed in yet. Will add them after the components and styling are all cleared. Happy to swap them for another system though. Hadn't seen facepaint before.

Reply:
Just swapped this for facepaint 🤡 493a456